src/thread.c

```diff
@@ -17,6 +17,7 @@
 #include "luv.h"
 
 #include "lthreadpool.h"
+#include <lualib.h>
```

Comment:
Should we move this include to luv.h alongside the lua.h and luaxlib.h includes?